src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/ElectrifiedCrucible.java

```diff
@@ -1,5 +1,7 @@
 package io.github.thebusybiscuit.slimefun4.implementation.items.electric.machines;
 
+import io.github.thebusybiscuit.slimefun4.api.MinecraftVersion;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
```

Comment:
M i s o r g a n i z a t i o n .

Reply:
How are they misorganized?
You wanted them in alphabetical order, they are.

Reply:
They aren't together with the other imports from `io.github.thebusybiscuit.slimefun4.`